src/SeleniumLibrary/__init__.py

```diff
@@ -580,9 +580,10 @@ class SeleniumLibrary(DynamicCore):
     languages, it only provides easy way to create full list keywords and their documentation in correct
     format. It is also possible to add keywords from library plugins by providing `--plugings` arguments
     to command. Example: `rfselib translation --plugings myplugin.SomePlugin /path/to/translation.json` The
-    genered json file contains `sha256` key, which constains the sha256 sum of the library documentation,
-    the sha256 sum is used by `rfselib translation --compare /path/to/translation.json` command, which compares
-    transation to to library and prints outs a table which tell if there are changes needed for translation file.
+    generated json file contains `sha256` key, which contains the sha256 sum of the library documentation.
+    The sha256 sum is used by `rfselib translation --compare /path/to/translation.json` command, which compares
+    the translation to the library and prints outs a table which tells if there are changes needed for
+    translation file.

```

Comment:
"... changes needed for **the** translation file."

Reply:
Also corrected